.size-limit.cjs

```diff
@@ -0,0 +1,11 @@
+module.exports = [
+  {
+    path: 'packages/common/dist/index.js',
+  },
+  {
+    path: 'packages/player-component/dist/dotlottie-player.mjs',
+  },
+  // {
+  //   path: 'packages/react-player/dist/index.js',
```

Comment:
@theashraf Curious why React will be ignored?

Reply:
@samuelOsborne @afsalz because we are using React 16 as dev dependency which is not compatible with the size-limit webpack 5 latest config, It works fine if the react dev dependency installed is react 18.

I've tested it on this PR, and you can have a look as well

https://github.com/dotlottie/player-component/pull/188